### DISCUSSION TITLE
[Art competition] Bottom-up Enumeration

### DISCUSSION THREAD

**Discussion by Gimongjin** [prosyslab-classroom/cs424-program-reasoning#261] [Art competition]
![DALL·E 2022-11-30 23 39 00 -  A student making bead necklace using lego blocks that has various sizes](https://user-images.githubusercontent.com/79095967/204824717-39074983-75e5-4337-8b37-665d6366d89b.png)

## Short Description
For me, Bottom-up Enumeration means A student making bead necklace using lego blocks that has various sizes".

## Long Description (up to 3 sentences)
Bottom-up Enumeration is a task to make a beautiful program using small beads - like a bead necklace. But this task does not uses pretty beads. Instead, it uses some variables and terminals looks like lego blocks.

**Comment by KihongHeo** [comment]
Make sense.
